backend/src/main/kotlin/com/tngtech/apicenter/backend/domain/handler/SessionHandler.kt

```diff
@@ -13,23 +13,14 @@ class SessionHandler constructor(
 
     fun authenticate(username: String, password: String): User? {
         val searchUser = externalAuthenticator.authenticate(username, password) ?: return null
+        // A new uuid is assigned on every authentication
```

Comment:
That comment makes no sense. Where does the UUID come from?

Reply:
True, it is an artefact of when a User's ID was a UUID, and I forgot to change it. (Now I see why people say "code never lies, comments sometimes do")